mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/MongoDBProperties.java

```diff
@@ -57,7 +57,9 @@
 
 	/**
 	 * The timeout used at the connection to the MongoDB instance. This value is set in milliseconds. Defaults to 5000.
+	 * @deprecated Use {@code hibernate.ogm.mongodb.driver.connectTimeout} instead.
 	 */
+	@Deprecated
```

Comment:
Should we remove it right away? It's going to be a new major version, so this sort of breakage seems acceptable to me.